exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -305,7 +306,14 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
           res.operation.context.requestPolicy !== 'cache-only'
         );
       }),
-      map(res => addCacheOutcome(res.operation, 'miss'))
+      map(res => {
+        dispatchDebug({
+          type: 'graphcacheMiss',
```

Comment:
Wondering if we need to generalise this more `cacheMiss` (because we also do just `cacheHit`)

Reply:
```suggestion
          type: 'cacheMiss',
```

Seems about right - both the [default cache](https://github.com/FormidableLabs/urql/pull/608/files#diff-6c7d190a26cd6cac287789abab335c32R70) and graphCache dispatch the same "event type"